packages/app/components/Queue/TA/TAQueueDetailButtons.tsx

```diff
@@ -84,6 +85,20 @@ export default function TAQueueDetailButtons({
     changeStatus(OpenQuestionStatus.Helping);
     openTeams();
   };
+  const deleteQuestion = async () => {
+    await changeStatus(
+      question.status === OpenQuestionStatus.Drafting
+        ? ClosedQuestionStatus.DeletedDraft
+        : LimboQuestionStatus.TADeleted
+    );
+    await API.questions.notify(question.id);
+  };
+
+  useHotkeys("shift+d", () => {
+    if (isCheckedIn) {
+      deleteQuestion();
+    }
```

Comment:
note to self: i think it needs the dependency array to include `isCheckedIn` and/or `question`?

Reply:
i was right